Character/CharacterStats.cpp

```diff
@@ -944,6 +948,14 @@ void CharacterStats::decrease_total_phys_dmg_mod(const int decrease) {
     remove_multiplicative_effect(phys_dmg_buffs, decrease, total_phys_dmg_mod);
 }
 
+void CharacterStats::increase_total_threat_mod(const int increase) {
+    add_multiplicative_effect(phys_dmg_buffs, increase, total_threat_mod);
```

Comment:
It should be `threat_buffs` that gets passed here instead of `phys_dmg_buffs` (same below).